main.go

```diff
@@ -108,6 +108,9 @@ func main() {
 	// e.PUT("/object/cross-bucket/copy", auth.Authorize(bh., writers...))
 	// e.PUT("/prefix/cross-bucket/copy", auth.Authorize(bh., writers...))
 
+	//auth
+	e.GET("/check_user_permission", auth.Authorize(bh.HandleCheckS3UserPermission, allUsers...))
```

Comment:
Not sure the purpose of this endpoint, but should allUsers really be able to check other users permissions? I would think you would want to limit to admins or something?

Reply:
Thanks!